README.md

```diff
@@ -10,13 +10,13 @@ This is the [Go](http://golang.org) client library for
 instrumenting application code, and one for creating clients that talk to the
 Prometheus HTTP API.
 
-__This library requires Go1.18 or later.__
```

Comment:
Hm, we sudden change to support Go 1.17? I think Go 1.18 or even Go 1.19 is fine? 🤔

Reply:
https://github.com/prometheus/client_golang/issues/1273